nemo/collections/tts/models/talknet.py

```diff
@@ -136,11 +135,11 @@ def _metrics(self, true_f0, true_f0_mask, pred_f0_sil, pred_f0_body):
         sil_acc *= true_f0_mask.type_as(sil_acc)
         sil_acc = sil_acc.sum() / true_f0_mask.sum()
 
-        body_mse = F.mse_loss(pred_f0_body, (true_f0 - self.F0_MEAN) / self.F0_STD, reduction='none')
+        body_mse = F.mse_loss(pred_f0_body, (true_f0 - self.f0_mean) / self.f0_std, reduction='none')
```

Comment:
is self.f0_mean and self.f0_std defined?

Reply:
Yes, in init.